packages/nimble-components/src/table/specs/table-column-specs/table-column-elapsed-time-hld.md

```diff
@@ -0,0 +1,177 @@
+# Elapsed Time Table Column
+
+## Overview
+
+The elapsed time table column will provide a way to visualize numeric data that represents a timespan. A 'timespan' is simply a numeric value representing a total number of seconds (i.e. a value of 5 means '5 seconds').
+
+### Background
+
+[Work Item](https://ni.visualstudio.com/DevCentral/_workitems/edit/2526116)
+
+### Non-goals
+
+-   APIs to configure individual aspects of the display (i.e. fractional digits for seconds display)
+-   APIs to format the elapsed time in a variety of ways
+    -   Show elapsed time as digital display (00:01:20.323)
+    -   Show elapsed time with symbols indicating a quantitative value (i.e. "< 1 sec")
+
+### Risks and Challenges
+
+Formatting the elapsed time units in the exact way that SLE presents them would require custom logic that differs from what the browser provides natively using APIs like `Intl.Numberformat.format()`. The preference here is to align with current browser behaviors for unit formatting, which includes localization.
+
+### Prior Art/Examples
+
+![ElapsedTime SLE](../spec-images/ElapsedTimeSLE.PNG)
+
+## Design
+
+Below is an example of how the `nimble-table-column-elapsed-time` would be used within a `nimble-table`:
+
+```HTML
+<nimble-table>
+    <nimble-table-column-elapsed-time field-name="elapsedTime">Elapsed Time</nimble-table-column-elapsed-time>
+</nimble-table>
+```
+
+The displayed value for elapsed time will always take the form of "\<hour display>, \<minute display>, \<second display>.\<fractional seconds>", where the commas are non-localized delimiters <sup>1</sup>.
+
+1. When and if we are able to adopt the `Intl.DurationFormat` to format values, we will no longer be hard-coding the commas in the string result as that object will produce the entirety of the display text, which may not match what we produce in our implementation.
+
+### API
+
+_*Props/Attrs*_
+
+-   `field-name`: string
+
+This API would result in a column whose value would only show up to the 'hours' unit (i.e. it would show hours, minutes, and seconds). We would show up to two fractional digits as needed. Formatting to change slightly from what SLE offers now using "hr" instead of "h", and "sec" instead of "s".
+
+Examples:
```

Comment:
The examples illustrate some of this but it might be worth making it explicit: larger units won't be displayed if their value is 0 but smaller ones will (I'm guessing).

**Not:**
- 0 hr, 0 min, 2 sec
- 1 hr, 3 sec
- 1 hr

**Yes:**
- 2 sec
- 1 hr, 0 min, 3 sec
- 1 hr, 0 min, 0 sec

Reply:
So that's now how the current `ElapsedTimePipe` is implemented. A value for a particular time unit is only included in the display if it has a non-zero value.

Reply:
I think I'm fine with that as a default. Thanks for updating the examples.

It does introduce a corner case for the value 0. Would we display blank? 0 sec?

Reply:
I've updated the example to just use "0 sec" instead of "2 sec" to demonstrate that we _will_ show a value for 0.